step-certificates/templates/ca.yaml

```diff
@@ -70,6 +102,9 @@ spec:
             allowPrivilegeEscalation: false
           {{- end }}
           volumeMounts:
+          - mountPath: /etc/ssl/certs/custom-trusted-ca.crt
+            name: custom-trusted-ca
+            subPath: ca-bundle.crt
```

Comment:
Looking at the PR, it looks like it touches 23 files, but most of the code is already in master, so it looks like a few rebases is missing.

In any case this is one of the change that I noticed. Instead  of hardcoding this, it might make more sense to have this in and the change in volume should be able to be configured in a values.yaml. The default values can be like:

```yaml
# Configure extra volumes and volume mounts as YAML expressions.
volumes: false
volumeMounts: false
```
And use something like this in this file, I might be wrong about the indentations:
```yaml
           volumeMounts:
           # ...
           {{- with .Values.volumeMounts }}
           # Extra volumeMounts
           {{ toYaml . | indent 10 }}
           {{- end }}
           # ...
         volumes:
         # ...
         {{- with .Values.volumes }}
         # Extra volumes
         {{ toYaml . | indent 8 }}
         {{- end }}
```